docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -263,11 +263,33 @@ spec:
   storageClassName: ''
   tenants:
     mode: openshift-logging
+  limits:
+    global:
+      ingestion:
+        ingestionBurstSize: 9 <1>
+        ingestionRate: 5 <2>
 ----
+<1> value in MiB (per push event)
+<2> value in MiB/s
 
 A dictionary holding the `.spec` for the LokiStack resource.
 
+The component configures fluentd as the default log forwarder.
+The default chunk size limit in fluentd is 8 MiB which the burst size limit of 9 MiB accounts for.
+The ingestion rate defines the MiB/s limit for a tenant.
+OpenShift Logging uses the following three tenants:
+* application
+* audit
+* infrastructure
```

Comment:
You want empty lines around the list, otherwise it won't get rendered correctly

```suggestion
OpenShift Logging uses the following three tenants:

* application
* audit
* infrastructure

```